app/resources/app-base.html.template

```diff
@@ -14,7 +14,13 @@
 {% block title %}{{block.super}}{% endblock title %}
 
 {% block logo %}
-  {% if not env.amp %}
+  {% if env.amp %}
+    <div class="header-space">
```

Comment:
We use word "header" for the entire top bar:
https://screenshot.googleplex.com/tqDHBhKmAa7
So it's a bit confusing to use the word for this DIV which only contains the button.

How about calling it e.g., title-bar-button-container, to be consistent with the class name title-bar-button used in non-AMP version?

Reply:
Rewtite this to more consistent name you suggested.